README.md

```diff
@@ -79,27 +82,37 @@ get_taxa(birds)
 ```
 
 ```
-##  [1] "Struthioniformes" "Tinamiformes"     "Craciformes"     
-##  [4] "Galliformes"      "Anseriformes"     "Turniciformes"   
-##  [7] "Piciformes"       "Galbuliformes"    "Bucerotiformes"  
-## [10] "Upupiformes"      "Trogoniformes"    "Coraciiformes"   
-## [13] "Coliiformes"      "Cuculiformes"     "Psittaciformes"  
-## [16] "Apodiformes"      "Trochiliformes"   "Musophagiformes" 
-## [19] "Strigiformes"     "Columbiformes"    "Gruiformes"      
-## [22] "Ciconiiformes"    "Passeriformes"
+## Source: local data frame [23 x 5]
+## 
+##      otu            label about xsi.type  otus
+##    <chr>            <chr> <chr>    <lgl> <chr>
+## 1    ou1 Struthioniformes  #ou1       NA   os1
+## 2    ou2     Tinamiformes  #ou2       NA   os1
+## 3    ou3      Craciformes  #ou3       NA   os1
+## 4    ou4      Galliformes  #ou4       NA   os1
+## 5    ou5     Anseriformes  #ou5       NA   os1
+## 6    ou6    Turniciformes  #ou6       NA   os1
+## 7    ou7       Piciformes  #ou7       NA   os1
+## 8    ou8    Galbuliformes  #ou8       NA   os1
+## 9    ou9   Bucerotiformes  #ou9       NA   os1
+## 10  ou10      Upupiformes #ou10       NA   os1
+## ..   ...              ...   ...      ...   ...
```

Comment:
This change seems OK to me since the README wasn't saying that the output is going to be a list. And presumably everyone can see from the output how to get the list.